crypto/crypto/src/traits.rs

```diff
@@ -247,3 +266,19 @@ pub trait Genesis: PrivateKey {
     /// Produces the genesis private key.
     fn genesis() -> Self;
 }
+
+/// A pub(crate) mod hiding a Sealed trait and its implementations, allowing
+/// us to make sure implementations are constrained to the crypto crate.
+// See https://rust-lang.github.io/api-guidelines/future-proofing.html#sealed-traits-protect-against-downstream-implementations-c-sealed
```

Comment:
Is this // (just comment and not in the docs) on purpose?

Reply:
Yes! I felt the api guidelines book are not yet in such a good state that we can refer to them in our public doc.